server/graphql/v2/interface/Account.js

```diff
@@ -274,6 +274,10 @@ const accountFieldsDefinition = () => ({
         type: new GraphQLList(GraphQLString),
         description: 'Filter on given types (creditcard, giftcard...)',
       },
+      service: {
+        type: new GraphQLList(GraphQLString),
```

Comment:
Shouldn't we use the `PaymentMethodService` enum?

Reply:
@Betree : Yep, that makes sense. I've changed that to use the `PaymentMethodService` enum. 👍🏽

Reply:
Should still be a GraphQLList.